src/main/java/com/iri/dao3/Application.java

```diff
@@ -18,5 +22,40 @@ public static void main(String[] args) {
         updatedManufacturer.setId(manufacturer.getId());
         manufacturerService.update(updatedManufacturer);
         manufacturerService.delete(manufacturer.getId());
+        Manufacturer manufacturerOne = new Manufacturer("Volvo", "Shwiden");
+        manufacturerService.create(manufacturerOne);
+
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver driverOne = new Driver("Leo", "8383");
+        Driver driverTwo = new Driver("Sanny", "1111");
+        Driver driverThree = new Driver("Goll", "2221");
+        driverService.create(driverOne);
+        driverService.create(driverTwo);
+        driverService.create(driverThree);
+        driverService.get(driverOne.getId());
+        driverService.getAll();
+        Driver updatedDriver = new Driver("Loe", "555");
+        updatedDriver.setId(driverOne.getId());
+        driverService.update(updatedDriver);
```

Comment:
```suggestion
        Driver updatedDriverOne = driverService.get(driverOne.getId());
        updatedDriverOne.setName("new name");
        driverService.update(updatedDriverOne);
```
this is a more appropriate use of update() method

Reply:
done